scalding-core/src/main/scala/com/twitter/scalding/Config.scala

```diff
@@ -298,6 +298,29 @@ trait Config extends Serializable {
   def setReducerEstimators(clsList: String): Config =
     this + (Config.ReducerEstimators -> clsList)
 
+  /**
+   * configure flow listeneres for observability
+   */
+  def addFlowListener[T](cls: Class[T]): Config =
```

Comment:
Can we put a better type on this? Is there no interface/root class for flow listeners?